src/neuroconv/tools/hdmf.py

```diff
@@ -62,10 +75,10 @@ def _get_default_buffer_shape(self, buffer_gb: float = 1.0) -> Tuple[int]:
                 buffer_bytes *= chunks_on_axis
                 padded_buffer_shape[axis] = self.maxshape[axis]
             else:  # Found an axis that is too large to use with the rest of the buffer; calculate how much can be used
-                k3 = np.floor(target_buffer_bytes / buffer_bytes)
+                k3 = int(math.floor(target_buffer_bytes / buffer_bytes))
```

Comment:
```suggestion
                k3 = math.floor(target_buffer_bytes / buffer_bytes)
```

`math.floor` returns a proper `int` type already